src/lib/journey/callbacks/choice/choice.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
   import type { ChoiceCallback } from '@forgerock/javascript-sdk';
 
+  import Radio from '$components/compositions/radio/animated.svelte';
   import Select from '$components/compositions/select-floating/floating-label.svelte';
 
   export let callback: ChoiceCallback;
   export let firstInvalidInput: boolean;
   export let idx: number;
+  export let displayType: 'radio' | 'select' = 'select';
```

Comment:
Given that we have decided (for now at least) not to react to the content in the stage parameter that drives this in a journey / tree, I've hard coded this to `select`. I'm fine with this, as this may be a capability we wish to introduce later, however for now I'm struggling to understand how we should leave this with regards to testing and review in Storybook / Chromatic. We can't effectively specify `radio` or `select` so all we will see at review is the effect of `select`. Hope this makes sense - @cerebrl any ideas?

Reply:
Yeah, let's go ahead and fully implement and test this `displayType` as a feature. Let's just keep it as a manual option that can be set in a custom "named" stage component. We just won't wire it up to the Stage property coming from AM until we settle on our response to the new news. 

So, for Storybook, we will have the Choice item with "Base" as the select type (since it is the default value), and then we will have a variant that uses the radio type. You can have a super simple Storybook interaction test like we have on the ConfirmationCallback: https://github.com/cerebrl/forgerock-web-login-framework/blob/main/src/lib/journey/callbacks/confirmation/confirmation.stories.js#L43.

As for e2e tests, I'd create a new journey by copying `LoginWithConfirmation` and adding a Choice node to it and rename it. Then modify this test by pointing it to the new journey and accommodating the new node/callback: https://github.com/cerebrl/forgerock-web-login-framework/blob/main/tests/widget/modal/widget-modal.login-with-confirm.test.js (rename it too). I say that because I want to avoid too many shallow e2e tests hitting our tenant in parallel. Just adding a node should be a more stable way to include it in our e2e without exponentially increasing the stress on the server.

Reply:
I've been looking at Storybook and am part of the way there I think.. We don't have an interaction test for "Base" choice i.e. select, so do we want an interaction for each of these?

Reply:
For Storybook Interactions, we need to test anything that contains some sort of view/rendering related logic, or if the component interfaces with another module, like the SDK. If the component just plainly renders something, then no need to do an interaction test on that, since there's no "interaction" per se (Testing whether some just renders is testing whether Svelte works, and we can trust that Svelte has its own tests for that).

Since this component does interact with the SDK, the minimal test is just answering the question of whether changing the value in the DOM input also changes the value in the expected callback `input` field. That's all, especially when the component is so simple. No elaborate test is needed.

Reply:
Understood. I've added one for "base" and another for "radio", given we have two variants for choice both interacting with the SDK. Thanks